public/contribute.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "portal-alpha",
+  "description": "File Sharing Experiment",
```

Comment:
Not sure how accurate the `name` and `description` fields are, but I did a copy-pasta from /package.json.